scripts/tests/test_validate_unwanted_patterns.py

```diff
@@ -0,0 +1,443 @@
+        "fd, expected",
+        [
+            (
+                io.StringIO(
+                    """
+with pytest.raises(ValueError):
+    pass
+""".strip()
+                ),
+                [
+                    (
+                        1,
+                        (
+                            "Bare pytests raise have been found. "
+                            "Please pass in the argument 'match' "
+                            "as well the exception."
+                        ),
+                    ),
+                ],
+            ),
+            (
+                io.StringIO(
+                    """
+with pytest.raises(ValueError, match="foo"):
+    with pytest.raises(ValueError):
+        pass
+    pass
+""".strip()
+                ),
+                [
+                    (
+                        2,
+                        (
+                            "Bare pytests raise have been found. "
+                            "Please pass in the argument 'match' "
+                            "as well the exception."
+                        ),
+                    ),
+                ],
+            ),
+            (
+                io.StringIO(
+                    """
+with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="foo"):
+        pass
+    pass
+""".strip()
+                ),
+                [
+                    (
+                        1,
+                        (
+                            "Bare pytests raise have been found. "
+                            "Please pass in the argument 'match' "
+                            "as well the exception."
+                        ),
+                    ),
+                ],
+            ),
+            (
+                io.StringIO(
+                    """
+with pytest.raises(
+    ValueError
+):
+    pass
+""".strip()
+                ),
+                [
+                    (
+                        1,
+                        (
+                            "Bare pytests raise have been found. "
+                            "Please pass in the argument 'match' "
+                            "as well the exception."
+                        ),
+                    ),
+                ],
+            ),
+            (
+                io.StringIO(
+                    """
+with pytest.raises(
+    ValueError,
+    # match = "foo"
+):
+    pass
+""".strip()
+                ),
+                [
+                    (
+                        1,
+                        (
+                            "Bare pytests raise have been found. "
+                            "Please pass in the argument 'match' "
+                            "as well the exception."
+                        ),
+                    ),
+                ],
+            ),
+        ],
+    )
+    def test_bad_pytest_raises(self, fd, expected):
+        result = list(validate_unwanted_patterns.bare_pytest_raises(fd))
+        assert result == expected
```

Comment:
Since this is the same as the previous, better to just have just  one test, merging the parameters of both. And you don't need the class anymore, since there is a single test in it.